recipe/meta.yaml

```
@@ -1,7 +1,7 @@
 {% set version = "5.0.0" %}
 {% set sha256 = "019f23c2192df793ac746595e94a403908749f8e0c484b403476d2611dd20970" %}
 
-{% set build_number = "1004" %}
+{% set build_number = "5" %}
```

As the version is the same it's safer to increment it:
```suggestion
{% set build_number = "1005" %}
```

(As it's a new variant that will be used when pinned it's probably safe to use a lower one, but there is no benefit from risking it and will only cause confusion.)